aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -48,8 +61,10 @@ module aptos_token::property_map {
         values: vector<vector<u8>>,
         types: vector<String>
     ): PropertyMap {
+        assert!(vector::length(&keys) <= MAX_PROPERTY_MAP_SIZE, error::invalid_argument(EPROPERTY_NUMBER_EXCEED_LIMIT));
         assert!(vector::length(&keys) == vector::length(&values), error::invalid_state(EKEY_COUNT_NOT_MATCH_VALUE_COUNT));
```

Comment:
nit: I think these should use error::invalid_argument instead of invalid_state, here and in the next line. IMO invalid_state refers to the current state of the resources.